### DISCUSSION TITLE
Does lightning-flash supports meta-learning process for text task?

### DISCUSSION THREAD

**Discussion by wenlai-lavine** [PyTorchLightning/lightning-flash#1226] [Q&A]
Thanks for the awesome work. there is an example for [image classification](https://github.com/PyTorchLightning/lightning-flash/blob/master/flash_examples/integrations/learn2learn/image_classification_imagenette_mini.py) task for meta-learning process, but I didn't find any examples in ```text``` tasks. Do anyone have some experience in using meta-learning on text task? such as, text classification etc.

**Comment by ethanwharris** [comment]
Hi @lavine-lmu we don't currently support meta learning for any task other than image classification, but please feel free to open a feature request for it :smiley:
